### ISSUE TITLE
Stop Duplicating Redis Data in Vets-api to the Secondary Redis

### ISSUE THREAD

**Issue by omgitsbillryan** [department-of-veterans-affairs/va.gov-team#11286]
## The Problem
We are still duplicating vets-api redis data for both dev & production. We want to undo this now that we have successfully migrated to the `vagov` Redis elastiache.

## Extra Context
Duplicating redis data was done as a way of migrating all of the vets-api redis data from `vetsgov` to `vagov` (a.k.a. "VAEC")

We basically want to undo everything that was done in this PR - https://github.com/department-of-veterans-affairs/vets-api/pull/4210

This can be thought of as the "clean up" part of https://github.com/department-of-veterans-affairs/va.gov-team/issues/9848

## Work to be Done
- [ ] Delete the duplication logic that lives in `vets-api` (see `initializers/01_redis.tb`) - [link](https://github.com/department-of-veterans-affairs/vets-api/blob/master/config/initializers/01_redis.rb#L7-L34)
- [ ] Remove all occurances of `Settings.redis.app_data.secondary_url`
  - [x] [dev](https://github.com/department-of-veterans-affairs/devops/blob/master/ansible/deployment/config/vets-api/dev-settings.local.yml.j2#L136)
  - [ ] [production](https://github.com/department-of-veterans-affairs/devops/blob/master/ansible/deployment/config/vets-api/prod-settings.local.yml.j2#L147)
  - [ ] [`settings.yml`](https://github.com/department-of-veterans-affairs/vets-api/blob/master/config/settings.yml#L414)

**Comment by alexpappasoddball** [comment]
@omgitsbillryan can this one be closed?

**Comment by omgitsbillryan** [comment]
@alexpappasoddball Nope. 

Just took a quick look and we are still duplicating data in `prod` across 2 redis elasticache instances 🤦 . Looks like [I got cold feet when I attempted this back in August](https://github.com/department-of-veterans-affairs/devops/pull/7357), since it wasn't clear that sidekiq wasn't still using this old redis instance.

Probably want to revisit this at some point lest we duplicate redis calls forever!
